src/bun.js/api/ffi.zig

```diff
@@ -444,6 +444,13 @@ pub const FFI = struct {
                 return error.DeferredErrors;
             }
 
+            for (this.symbols.map.values()) |symbol| {
```

Comment:
Let's make this a pointer to avoid an extra unnecessary copy
```suggestion
            for (this.symbols.map.values()) |*symbol| {
```